app/views/devise/registrations/edit.html.erb

```diff
@@ -47,17 +47,17 @@
 
     <div class="registration__row">
       <div class="registration__field">
-        <%= f.form_row :password do %>
-          <%= f.errors_for :password %>
-          <%= f.label :password, class: "form__label-heading" %>
-          <%= f.password_field :password, autocomplete: "off", "aria-describedby" => "tooltip_password" %>
+        <%= f.form_row :contact_number do %>
+          <%= f.errors_for :contact_number %>
+          <%= f.label :contact_number, t('authentication.settings.contact_number.label'), class: "form__label-heading" %>
```

Comment:
is the form object a User? if so you should be able to use this https://github.com/moneyadviceservice/frontend/blob/update-edit-form-6613/config/locales/en.yml#L104